stdlib/fun.mli

```diff
@@ -0,0 +1,33 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                         The OCaml programmers                          *)
+(*                                                                        *)
+(*   Copyright 2018 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Functions and operators on functional values.
+
+    @since 4.08 *)
+
+module Ops : sig
+  val ( %< ) : ('a -> 'b) -> ('c -> 'a) -> 'c -> 'b
+  (** Function composition: [f %< g] is equivalent to [fun x -> f (g x)].
+      Left-associative at precedence level 7/11.
+      @since 4.08
+  *)
+
+
+  val ( %> ) : ('a -> 'b) -> ('b -> 'c) -> 'a -> 'c
+  (** Function reverse-composition: [f %> g] is equivalent to [fun x -> g (f x)].
```

Comment:
Travis is [failing](https://travis-ci.org/ocaml/ocaml/jobs/445057742#L486) because this line is over 80 columns.

Reply:
Btw. it would be nice to have a toplevel target like `make pr-check` and be documented [here](https://github.com/ocaml/ocaml/blob/trunk/CONTRIBUTING.md#run-tests-before-sending-a-pr) that runs most of these CI checks to avoid too many suffering round trips with the CI.